e2e-tests/portal/cypress/e2e/journeys/login/navigating-using-sign-in-link.spec.js

```diff
@@ -140,6 +140,21 @@ context('navigating using sign in link', () => {
       signInLink.visit({ token: NOT_EXPIRED_SIGN_IN_TOKEN.signInTokenFromLink, userId: bank1Maker1Id }, { failOnStatusCode: false });
       signInLink.shouldDisplayAccountSuspended();
     });
+
+    it('Opening a valid sign in link and logging in but then disabling the user and visiting the landing page will redirect the user to the login page', () => {
```

Comment:
this tests disabling a user once they are logged in happily and making sure they get logged out